src/commands/account/use.ts

```diff
@@ -0,0 +1,77 @@
+import fs from "fs-extra";
+import chalk from "chalk";
+import Command from "../../base";
+import { prompt } from "inquirer";
+import { CLIError } from "@oclif/errors";
+import { flags } from "@oclif/command";
+import { GLOBAL_CONF_PATH } from "../../constants";
+
+interface IAccount {
+  email: string;
+  api_token: string;
+  region: string;
+}
+
+interface IAccounts {
+  [key: string]: IAccount;
+}
+
+interface ILiaraJson {
+  api_token?: string;
+  region?: string;
+  current?: string;
+  accounts?: IAccounts;
+}
+
+export default class AccountUse extends Command {
+  static description = "select an account";
+
+  static flags = {
+    ...Command.flags,
+    name: flags.string({ char: "n", description: "account name" }),
+  };
+
+  async run() {
+    const { flags } = this.parse(AccountUse);
+    const liara_json: ILiaraJson = this.gatherLiaraJson();
+    if (!liara_json || !liara_json.accounts || Object.keys(liara_json.accounts).length === 0) {
+      this.error("Please add acount via 'liara account:add' command.");
+    }
+    const name = flags.name ? flags.name : await this.promptName();
+    const selectedAccount = liara_json.accounts[name];
+    !Boolean(selectedAccount) &&
+      this.error(`account-name ${name} is not exist!`);
```

Comment:
I guess this is a better error message:
`Could not find any account associated with this name ${name}.`